src/store/ChartStore.js

```diff
@@ -247,7 +248,15 @@ class ChartStore {
                 } else {
                     this.changeSymbol(this.defaultSymbol);
                 }
+
                 this.setLayoutData(context);
+                /**
+                 * Updating market close status each 10 minute
+                 */
+                this.updateMarketClosedStatus(api);
```

Comment:
if we are going to `api` all over the class, there's no need to pass it around as function argument. Just save it as `this.api`.